src/main/java/org/sonarsource/sonarlint/ls/settings/SettingsManager.java

```diff
@@ -156,6 +165,20 @@ public void didChangeConfiguration() {
         notifyListeners(newWorkspaceSettings, oldWorkspaceSettings, newDefaultFolderSettings, oldDefaultFolderSettings);
 
         resubscribeForServerEvents(oldWorkspaceSettings, newWorkspaceSettings, previousProjectKeysByConnectionId, currentProjectKeysByConnectionId);
+        var connections = getCurrentSettings().getServerConnections();
+        var params = backendServiceFacade.getInitParams();
+        var sqConnections = connections.entrySet().stream()
+          .filter(entry -> !entry.getValue().isSonarCloudAlias())
+          .map(it -> new SonarQubeConnectionConfigurationDto(it.getKey(), it.getValue().getServerUrl()))
+          .collect(Collectors.toList());
+        var scConnections = connections.entrySet().stream()
+          .filter(entry -> entry.getValue().isSonarCloudAlias())
+          .map(it -> new SonarCloudConnectionConfigurationDto(it.getKey(), it.getValue().getOrganizationKey()))
+          .collect(Collectors.toList());
+        params.setSonarQubeConnections(sqConnections);
+        params.setSonarCloudConnections(scConnections);
+        backendServiceFacade.initOnce();
+        backendServiceFacade.getBackend().didChangeConfiguration(connections);
```

Comment:
This looks a bit like the "feature envy" code smell. You get something from the facade, then you call 2 methods on it. To me it's a sign that some stuff could be moved to the facade. I was a bit confused at first by the successive calls to `initOnce` and `didChangeConfiguration`, and I think if all this code was located in the facade it would be clearer (if the backend is not yet initialized, then initialize it, else notify it about changes). Also the conversion code exists twice, here and in `didChangeConfiguration`